graphify-out/memory/query_20260831_041023_what_does__verified__mean_for_a_skills_integration.md

```
@@ -0,0 +1,71 @@
+outcome: "corrected"
+correction: "The overturned belief: that running the applicable gates and reporting their\nexit codes constitutes verification.\n\nIt does not. In this round `md-budget`, `skill-lint`, `lint-docs` and\n`mirror_drift` all returned 0 against a skills integration that was missing\nthree P1 behaviours, promised an advisory that cannot fire, and contained a\nchecklist row contradicting a rule two paragraphs above it. Every gate was\nhonest; none of them was asking the question.\n\nThree corollaries, each measured this round:\n\n1. A HOOK'S GREEN CAN DESCRIBE A TREE THAT NO LONGER EXISTS. `md_size_budget`\n   passed, then hk's `rumdl_format` reflowed the file in the same commit and\n   added a line, shipping 501 against a 500 cap. Run the formatter BEFORE the\n   measurement, and re-run any size gate against the COMMITTED tree.\n\n2. A DIFF REVIEW CANNOT SEE INCOMPLETENESS. It reviews what changed. When the\n   defect is that something was never written, only a full-content read against\n   the dependency's own source finds it.\n\n3. THE AUTHOR IS THE WRONG AUDITOR OF THEIR OWN FIX. Two fixes this round each\n   introduced a new defect, and one contradicted its own conditional rule inside\n   the same commit that created it.\n\nAnd the failure shape behind most of it: stating a claim slightly stronger than\nits source supports. Three instances in one round — a `~90 tokens` figure that\nmeasured 440, a \"the plugin recommends slug mode\" scoped in the source to\nparallel sessions only, and \"silently stops reaching context\" against a source\nthat echoes a notice. Each was caught externally; none by re-reading my own words.\n"
+---
+
+# Q: What does 'verified' mean for a skills integration, and what actually catches defects?
+
+## Answer
+
+# Round kb-20260830.003 — what it asked and what it found
+
+The round installed and integrated the `planning-with-files` plugin, landed PR
+#637, corrected GitHub issue #638's measured claims, and moved the mise pin.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)